routes/account.js

```diff
@@ -611,6 +784,14 @@ module.exports = function (
 
   if (isProduction) {
     delete routes[0].config.validate.payload.preVerified
+
+    for (var i = 0; i < routes.length; ++i) {
+      var route = routes[i];
+      if (route.path === '/account/lock') {
+        routes = routes.splice(i, 1)
```

Comment:
>  isProduction ? undefined : { ...the dev-only route... },

I just tried this approach, Hapi flipped its lid.

> I also quite like the way that @seanmonstar did it over in the oauth-server, where each route is defined in a separate file and we explicitly stitch together the required routes as a separate step

We take this approach in the content server too. It causes an explosion of files, but each file is very focused.